challenges/2021-04-06-intervening-verbiage/src/main/java/io/codeconnector/codedojo/InterveningVerbiage.java

```diff
@@ -38,6 +38,29 @@ public int countWordsBetween(
     ) throws IllegalArgumentException
     {
         // You should put the challenge solution here
+        String[] wordsArray = words.split(" ");
+        System.out.println(wordsArray);
+        int count = 0;
+        boolean counting = false;
+
+        for (String word : wordsArray) {
+            System.out.println(word);
+            if (word.equalsIgnoreCase(stop)){
+                return count;
+            }
+            if (counting==true) {
```

Comment:
Just want to provide feedback and give you a chance to work on committing to PR's when someone requests feedback. So, line 51 is a bad coding principal. The reason is, the variable `counting` is already equal to `true` because you assigned it that when you found your first word. So what your evaluation on line 51 is doing is:

```
if (true==true) {
...
}
```
We don't need to do that. Instead, we can simply do:
```
if (counting) {
 ...
}
```
I just wanted to offer this so you have a chance to see a real code review after your first awesome day. Thanks for participating!

Reply:
Nice